README.md

```diff
@@ -33,6 +33,20 @@ To make it easier to test Vapor apps locally on Linux using Docker, we have made
 - `Testing/Dockerfile-Swift4.1` which is using Swift 4.1 and is supposed to be used for Vapor 2 or Vapor 3 projects.
 - `Testing/Dockerfile-Swift4.2` which is using Swift 4.1 and is supposed to be used for Vapor 2 or Vapor 3 projects.
 
+## 🌳 Environment variables
+
+Environment variables are supported in Docker. Please read the [docs regarding environment variables](https://docs.docker.com/compose/environment-variables/) for info regarding how to configure them
```

Comment:
```suggestion
Environment variables are supported in Docker. Please read the [docs regarding environment variables](https://docs.docker.com/compose/environment-variables/) for info regarding how to configure them.
```

Reply:
Added the dot ;)

Reply:
Next time you can also just use the "commit suggestion" button 🙂